apkid/rules/elf/obfuscators.yara

```diff
@@ -372,6 +372,45 @@ rule dexguard_native : obfuscator
       and any of them
 }
 
+rule dexguard_native_arm64 : obfuscator
+{
+  meta:
+    description = "DexGuard (>= 9.0, arm64)"
```

Comment:
Not needed to describe the arch, it will be duplicated in the matched library. Better `description` = "DexGuard v9.x"